requirements.txt

```diff
@@ -1,5 +1,10 @@
 Django==1.11.2
+factory-boy==2.8.1
+Faker==0.7.17
+ipaddress==1.0.18
 mysqlclient==1.3.10
 page-objects==1.1.0
+python-dateutil==2.6.0
 pytz==2017.2
 selenium==3.4.3
+six==1.10.0
```

Comment:
Make sure `requirements.txt` only contains packages you directly use. Packages that are installed because they're requirements for the packages you have in your `requirements.txt` shouldn't be in this file.

For instance, `Faker` is installed because it's a requirement for the `factory-boy`, so `factory-boy` should define that requirement.

I suggest you manually change this file to control the content instead of relying on pip freeze (which is OK to use the first time you're generating this file).